packages/block-library/src/image/view.js

```diff
@@ -59,6 +59,19 @@ const { state, actions, callbacks } = store(
 				const { imageId } = getContext();
 				return state.metadata[ imageId ].imageButtonTop;
 			},
+			get isContentHidden() {
+				const ctx = getContext();
+				return (
+					state.overlayEnabled && state.currentImageId === ctx.imageId
```

Comment:
Not sure if it could be used elsewhere, but maybe in the future we could consider moving `state.currentImageId === ctx.imageId` to a new `isCurrentImage` derived state property?

It's true that right now, it is only used in `isContentHidden` and `isContentVisible` if I am not mistaken.

Reply:
> Not sure if it could be used elsewhere, but maybe in the future we could consider moving state.currentImageId === ctx.imageId to a new isCurrentImage derived state property?

That sounds good to me as a future enhancement. Right now the code is a little verbose with `isContentHidden` and `isContentVisible` right beside each other, but I think the intent is clear and it's easy to see the contrast between them without much fuss.